jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java

```diff
@@ -235,4 +237,77 @@ public void requestCreated( @Observes RequestChangedEvent event ) {
+                              v -> removeAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_ID,
+                                                                       v))
+        );
+
+        view.addTextFilter(constants.BusinessKey(),
+                           constants.FilterByBusinessKey(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_BUSINESSKEY,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_BUSINESSKEY,
+                                                                    v))
+        );
+
+        view.addTextFilter(constants.Type(),
+                           constants.FilterByType(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_COMMANDNAME,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_COMMANDNAME,
+                                                                    v))
+        );
+
+        view.addTextFilter(constants.Process_Description(),
+                           constants.FilterByProcessDescription(),
+                           v -> addAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_DESCRIPTION,
+                                                                 v)),
+                           v -> removeAdvancedSearchFilter(equalsTo(COLUMN_PROCESS_INSTANCE_DESCRIPTION,
+                                                                    v))
+        );
+
+        final Map<String, String> status = new HashMap<>();
+        status.put(RequestStatus.CANCELLED.name(),
+                   constants.Cancelled());
+        status.put(RequestStatus.DONE.name(),
+                   constants.Completed());
+        status.put(RequestStatus.ERROR.name(),
+                   constants.Error());
+        status.put(RequestStatus.QUEUED.name(),
+                   constants.Queued());
+        status.put(RequestStatus.RETRYING.name(),
+                   constants.Retrying());
+        status.put(RequestStatus.RUNNING.name(),
+                   constants.Running());
+
+        view.addSelectFilter(constants.Status(),
+                             status,
+                             false,
+                             v -> addAdvancedSearchFilter(equalsTo(COLUMN_STATUS,
+                                                                   v)),
+                             v -> removeAdvancedSearchFilter(equalsTo(COLUMN_STATUS,
+                                                                      v))
+        );
+
+        //TODO Missing process id and creation date
+
+        final FilterSettings settings = view.getAdvancedSearchFilterSettings();
+        final DataSetFilter filter = new DataSetFilter();
+        filter.addFilterColumn(equalsTo(COLUMN_STATUS,
+                                        RequestStatus.RUNNING.name()));
```

Comment:
@nmirasch that was my initial intention for both filters, the reason I only used one status is that the user can not create the same filter with multiple statuses (using OR) via the UI. I thought that would cause confusion. When adding new filters in the UI it always appends as an AND operation.